selfdrive/car/ford/carstate.py

```diff
@@ -29,9 +29,14 @@ def update(self, cp, cp_cam):
     self.unsupported_platform = (cp.vl["VehicleOperatingModes"]["TrnAinTq_D_Qf"] == 0 and
                                  self.CP.carFingerprint not in CANFD_CAR)
 
-    # Occasionally on startup, the ABS module recalibrates the steering pinion offset, so we need to block engagement
-    # The vehicle usually recovers out of this state within a minute of normal driving
-    self.vehicle_sensors_valid = cp.vl["SteeringPinion_Data"]["StePinCompAnEst_D_Qf"] == 3
+    if self.CP.carFingerprint in CAN_EDGE:
```

Comment:
any idea why the EDGE is different?

Reply:
I'm not sure why. I checked with other Ford drivers and they confirmed that the F-150's have no data at `ExtSteeringAngleReq2`. Likewise, my Edge has no data at `StePinCompAnEst_D_Qf` or `StePinComp_An_Est`. There's another Edge user that just bought a C3x. He has a different ABS fingerprint, the rest matches, and his data matches with mine. 'SteeringPinion_Data' shows up at a different ID in the Edges as well. I originally created a new DBC, changing ID from 7e to 85, but abandoned that once I found `StePinComp_An_Est` has no data. That changed CAN ID may indicate a larger change to the PSCM?

Reply:
I can confirm this is working with my Ford Edge Titanium 2022. Stock openpilot:master did not work after fingerprinting and required these extra steps. I can provide logs if they would be helpful.